api/services/email.service.js

```diff
@@ -0,0 +1,34 @@
+const nodeMailer = require("nodemailer");
+const dotenv = require("dotenv");
+
+dotenv.config();
+
+const transporter = nodeMailer.createTransport({
+  service: "gmail",
+  auth: {
+    user: process.env.EMAIL_USER,
+    pass: process.env.EMAIL_PASSWORD,
+  },
+});
+
+class EmailService {
+  constructor(transporter) {
+    this.transporter = transporter;
+  }
+
+  async sendConfirmationEmail(name, email, confirmationToken) {
+    const mailOptions = {
+      from: process.env.EMAIL_USER,
+      to: email,
+      subject: "Confirm your account",
+      html: `<h1>Organizer</h1>
```

Comment:
Lets use some template system for it. It is very small text but it will be good from learning prospective. (Lets use pug maybe)

Reply:
Used pug template to send emails